tool/jt.rb

```diff
@@ -2043,14 +2047,19 @@ def graph(*args)
       "--engine.CompileOnly=#{method}",
       '--engine.MultiTier=false',
       '--engine.NodeSourcePositions',
+      '--vm.Dgraal.FullUnroll=false',
+      '--vm.Dgraal.PartialUnroll=false',
+      '--vm.Dgraal.LoopPeeling=false',
+      '--vm.Dgraal.LoopUnswitch=false',
+      '--vm.Dgraal.OptScheduleOutOfLoops=false',
```

Comment:
I guess this is mainly to simplify the graph, but it might get farther and farther from what the actual compilation graph is.
For instance I think `FullUnroll` is something one would want to know when it happens. If someone based on the seafoam graph manually adds an ExplodeLoop specialization when the unroll would already be done it seems suboptimal.

`LoopPeeling` is likely inconvenient to explore the graph, but it might help hoisting guards out of the loop, etc.

I think these extra non-standard options should be opt-in with an extra option like `--simplify`.

Reply:
Will move into a different PR.